components/src/dynamo/profiler/utils/config_modifiers/protocol.py

```
@@ -843,3 +843,68 @@ def apply_dgd_overrides(dgd_config: dict, overrides: dict) -> dict:
+        if not isinstance(svc, dict) or svc_name in _OVERRIDE_NON_WORKER_SERVICES:
+            continue
+        extra_pod_spec = svc.get("extraPodSpec")
+        if not isinstance(extra_pod_spec, dict):
+            continue
+        main_container = extra_pod_spec.get("mainContainer")
+        if not isinstance(main_container, dict):
+            continue
+        args = main_container.get("args") or []
+        if _TRUST_REMOTE_CODE_FLAG in args:
+            continue
+        main_container["args"] = list(args) + [_TRUST_REMOTE_CODE_FLAG]
```

Ruff (0.15.15)

[warning] Consider `[*list(args), _TRUST_REMOTE_CODE_FLAG]` instead of concatenation

Replace with `[*list(args), _TRUST_REMOTE_CODE_FLAG]`

(RUF005)